sources/functions/ffmpeg

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# [Swizzin :: Install ffmpeg package]
+#
+# Author:   liara
+#
+# Copyright (C) 2017 Swizzin
+# Licensed under GNU General Public License v3.0 GPL-3 (in short)
+#
+#   You may copy, distribute and modify the software as long as you track
+#   changes/dates in source files. Any modifications to our software
+#   including (via compiler) GPL-licensed code must also be made available
+#   under the GPL along with build & install instructions.
+
+ffmpeg_install() {
+    export distribution=$(lsb_release -is)
+    export release=$(lsb_release -rs)
+    export codename=$(lsb_release -cs)
```

Comment:
@liaralabs can these go?

Reply:
Those are taken from the original install script. It's just simple refactoring. I disagree with making changes not related to the pull request. It's harder to track down regressions. We should remove with a new commit if required.

Reply:
Sure, why not